### PR TITLE
[NEW] debug timer

### PR DESCRIPTION
**Pull request by ericgibb**

### CODE REVIEW

File: src/debug.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+/* eslint-disable no-undef*/
+/* eslint-disable no-console*/
+
+global.timer = 0;
+global.timerStep = null;
+
+function start(l) {
+  const label = l || '';
+  const now = performance.now();
+  console.log(`${label + ' '}START: now`);
```
Comment (zachgibb) [inline src/debug.js]: did you mean `START: ${now}`?

Comment (ericgibb) [reply src/debug.js]: oh yeah, I can see how that's confusing. I'll fix this, but I don't think there's a use for the timestamp output there. it's just going to show either `0s` or the HRT in milliseconds... which... I think is more useful in the `step` and `stop` stuff

Comment (zachgibb) [reply src/debug.js]: ah, gotcha. yeah, that makes sense

**Comment by ericgibb** [comment]
image

**Pull request merged**